docs/cloud-storage-fileset-example.md

```diff
@@ -0,0 +1,678 @@
+---
+title: "How to use cloud storage fileset"
+slug: /how-to-use-cloud-storage-fileset
+keyword: fileset S3 GCS ADLS OSS
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document aims to provide a comprehensive guide on how to use cloud storage fileset created by Gravitino, it usually contains the following sections:
```

Comment:
End a sentence with a full period `.`; start a new sentence with a capital letter,
and optionally leave two spaces between two sentences.